app/assets/stylesheets/base/helpers.scss

```diff
@@ -36,6 +36,10 @@
   border-top: var(--su-7) solid var(--accent-brand);
 }
 
+.top-margin-4 {
+  margin-top: var(--su-4);
```

Comment:
I think there might be existing utility classes for things like margins and text colour? Although I am still not really sure what approach we're supposed to be taking between Crayons/Tailwind/custom css